src/containers/Ledgers/LedgerMetrics.tsx

```diff
@@ -0,0 +1,120 @@
+import { useTranslation } from 'react-i18next'
+import { Tooltip, useTooltip } from '../shared/components/Tooltip'
+import { renderXRP } from '../shared/utils'
+import PauseIcon from '../shared/images/ic_pause.svg'
+import ResumeIcon from '../shared/images/ic_play.svg'
+import './css/ledgerMetrics.scss'
+import { useIsOnline } from '../shared/SocketContext'
+import { useLanguage } from '../shared/hooks'
+
+const DEFAULTS = {
+  load_fee: '--',
+  txn_sec: '--',
+  txn_ledger: '--',
+  ledger_interval: '--',
+  avg_fee: '--',
+  quorum: '--',
+  nUnl: [],
+}
+
+export const LedgerMetrics = ({
+  data: suppliedData,
```

Comment:
I don't see where `suppliedData` is defined

Reply:
`data` is the props name but the linter prevents redefining a prop so i name it differently locally scoped to the function to make it shorter instead of having a longer name ever where i reference all the metrics